app/segments/drill/[id]/submission/input.js

```diff
@@ -136,7 +153,114 @@ async function uploadAttempt(
   }
 }
 
-//TODO: Create a function to check leaderboard and update if needed
+//A function to check leaderboard and update if needed
+function handleLeaderboardUpdate(
+  uploadData,
+  drillInfo,
+  currentLeaderboard,
+  teamId,
+  drillId,
+  userId,
+  queryClient,
+) {
+  const leaderboardData = currentLeaderboard;
```

Comment:
Why is this here?

Reply:
drillInfo.mainOutputAttempt would be the better candidate to be given a const, because it's used a lot and is relatively long

Reply:
I removed the `const leaderboardData = currentLeaderboard;` and added `const mainOutputAttempt = drillInfo.mainOutputAttempt;` as suggested